Source/monster.cpp

```diff
@@ -1126,13 +1126,17 @@ void MonsterAttackPlayer(Monster &monster, Player &player, int hit, int minDam,
 	hit += 2 * (monster.level(sgGameInitInfo.nDifficulty) - player._pLevel)
 	    + 30
 	    - ac;
-	int minhit = 15;
-	if (currlevel == 14)
-		minhit = 20;
-	if (currlevel == 15)
-		minhit = 25;
-	if (currlevel == 16)
+	int minhit;
+	switch (currlevel) {
+	case 16:
 		minhit = 30;

```

Comment:
Each `case` branch needs a `break;`

Reply:
Oh my god lmao
I've been doing python for the past 2 months. Already getting rusty on c++

Reply:
Alternativately, extract it into a function and replace assignment with `return`. Then there is no need for `break;`.

Reply:
I wouldn't mind doing somethings in Rust :)